packages/-ember-data/tests/unit/model-test.js

```diff
@@ -665,6 +665,41 @@ module('unit/model - Model', function(hooks) {
         assert.equal(eventMethodArgs[1], 2);
       }
     );
+
+    deprecatedTest(
```

Comment:
This test fails in the production build, likely because deprecations and assertions are removed there.

Looks like `deprecatedTest` needs to be tweaked to avoid emitting a test at all when `!DEBUG`:

https://github.com/emberjs/data/blob/60de9b60c31e484ff1302dda9b1b4caa1a995bf0/packages/-ember-data/tests/helpers/deprecated-test.js#L15

Reply:
swapped `deprecatedTest` for `testInDebug`